packages/components/pie-webc-core/src/test/decorators/safe-custom-element.spec.ts

```diff
@@ -36,6 +36,7 @@ describe('safeCustomElement', () => {
 
         // Act
         @safeCustomElement(tag)
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
         class TestElementB extends HTMLElement {
             static v = '1.0.0';
         }
```

Comment:
I'm trying to understand what's going on here. Do we still need the class if it's not being used?

Reply:
Ben is correct - one line above you'll see `@safeCustomElement(tag)` - this is a decorator that is in fact referencing the class it is decorating. So it needs to stay. The lint rule is just oblivious :)